tests/chainerx_tests/unit_tests/routines_tests/test_normalization.py

```diff
@@ -152,7 +152,7 @@ def generate_inputs(self):
         param_dtype = self.param_dtype
 
         x = numpy.random.uniform(-1, 1, x_shape).astype(x_dtype)
-        gamma = numpy.random.uniform(-1, 1, reduced_shape).astype(param_dtype)
+        gamma = numpy.random.uniform(0.5, 1, reduced_shape).astype(param_dtype)
```

Comment:
Gamma is the scaling factor which realistically is closer to 1. These configurations are identical to that of that of Chainer.